packages/integration-tests/src/relations/ManyToManyCollection.test.ts

```diff
@@ -31,7 +31,7 @@ describe("ManyToManyCollection", () => {
     expect(tag.books.get.length).toEqual(1);
     expect(book.tags.get[0]).toStrictEqual(tag);
     expect(tag.books.get[0]).toStrictEqual(book);
-    expect(em.__data.joinRows["books_to_tags"].length).toEqual(1);
```

Comment:
This was the only place outside of `orm/src` we were poking at the internal API, and it was just for an assertion, so went ahead and kept it with a cast.